File: ts/wam/2220.ts
export const WamBusinessToolsImpression = 2220 as const

export interface BusinessToolsImpressionEvent {
  /** field 1, wire `business_tools_session_id` */
  businessToolsSessionId?: string
  /** field 2, wire `business_tools_sequence_number` */
  businessToolsSequenceNumber?: number
  /** field 4, wire `business_tools_entry_point` */
  businessToolsEntryPoint?: BUSINESSTOOLSENTRYPOINTTYPE
}

File: go/wam/2220.go
const WamBusinessToolsImpression = 2220 // channel: regular

type BusinessToolsImpressionEvent struct {
	BusinessToolsSessionId string `wam:"business_tools_session_id"` // field 1
	BusinessToolsSequenceNumber int64 `wam:"business_tools_sequence_number"` // field 2
	BusinessToolsEntryPoint BUSINESSTOOLSENTRYPOINTTYPE `wam:"business_tools_entry_point"` // field 4
}